File: DisplaFrontend/src/app/model/precioLenteCliente.ts
import { Cliente } from './cliente';
import { PrecioLente } from './precioLente';

export interface PrecioLenteCliente {
    Id: number,
    IdPrecioLente: number,
    IdPrecioLenteNavigation: PrecioLente
    IdCliente: number,
    IdClienteNavigation: Cliente,
    Descuento: number,
    Especial: boolean
}

File: DisplaBackend/DisplaBackend/Models/PrecioLenteCliente.cs
using System;
using System.Collections.Generic;

namespace DisplaBackend.Models
{
    public partial class PrecioLenteCliente
    {
        public int Id { get; set; }
        public int IdPrecioLente { get; set; }
        public int IdCliente { get; set; }
        public double? Descuento { get; set; }
        public bool? Especial { get; set; }

        public virtual Cliente IdClienteNavigation { get; set; }
        public virtual PrecioLente IdPrecioLenteNavigation { get; set; }
    }
}
